obspy/io/gcf2mseed.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
+
+# convert GCF files to miniseed
+# by Ran Nof @ BSL, 2016
+# ran.nof@gmail.com
+
+from gcf.core import _read_gcf as read_gcf
+from gcf.core import _is_gcf as is_gcf
+from glob import glob,sys
+from obspy import Stream
+import argparse,textwrap
+
+# set the argument parser
+parser = argparse.ArgumentParser(
+         formatter_class=argparse.RawDescriptionHelpFormatter,
+         description='''Convert Guralpe Compressed Format (GCF) to Miniseed''',
+         epilog=textwrap.dedent('''\
+             See more details regarding GCF:
+             http://www.guralp.com/apps/ok?doc=GCF_Intro
+    
+             The script only converts data streams
+        
+             Created by Ran Novitsky Nof @ BSL, 2016'''))
+parser.add_argument('-o',metavar='Output_file',type=str,help='Output Miniseed file name.')
+parser.add_argument('gcf_file',nargs='+',type=str,help='input file(s) name.')
+parser.add_argument('-O',metavar='option',type=str,nargs='*',default=["'MSEED'"],help='''\
+                         Options for the obspy write command (default: MSEED.)
+                         Example option:
+                         -O "format='MSEED'" encoding=11 reclen=512 byteorder=1.
+                         Note the " and ' around the format and MSEED values.'''
+                    )
+
+def getStream(FileList):
+  st = Stream()
```

Comment:
Yep. Sorry. Forgot all about it.

On Thu, Jun 9, 2016 at 2:46 PM, Elliott Sales de Andrade <
notifications@github.com> wrote:

> In obspy/io/gcf2mseed.py
> <https://github.com/obspy/obspy/pull/1449#discussion_r66527854>:
>
> > +             http://www.guralp.com/apps/ok?doc=GCF_Intro
> > +
> > +             The script only converts data streams
> > +
> > +             Created by Ran Novitsky Nof @ BSL, 2016'''))
> > +parser.add_argument('-o',metavar='Output_file',type=str,help='Output Miniseed file name.')
> > +parser.add_argument('gcf_file',nargs='+',type=str,help='input file(s) name.')
> > +parser.add_argument('-O',metavar='option',type=str,nargs='*',default=["'MSEED'"],help='''\
> > +                         Options for the obspy write command (default: MSEED.)
> > +                         Example option:
> > +                         -O "format='MSEED'" encoding=11 reclen=512 byteorder=1.
> > +                         Note the " and ' around the format and MSEED values.'''
> > +                    )
> > +
> > +def getStream(FileList):
> > +  st = Stream()
>
> This file definitely did not go through flake8.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/obspy/obspy/pull/1449/files/974a93d3f7083dd0f186664e2bbff4b7dbe9c22e#r66527854>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKICcm6ZEnGiFP9OxqMpTyiVMz8upwgAks5qKIm-gaJpZM4IyR6B>
> .
>



-- 
Ran Novitsky Nof
Post-Doc Fellow
Berkeley Seismological  Laboratory
University of California Berkeley

Rm 209, McCone Hall
Mob: 1-805-418-0581
Web: http://ran.rnof.info